content/number-theory/Diophantine.h

```diff
@@ -0,0 +1,20 @@
+/**
+ * Author: Unknown
+ * Date: 2024-10-11
+ * Source: vasito
```

Comment:
```suggestion
 * Source: notebook el vasito
```
Así está puesto en el resto de los que vienen de el vasito